### ISSUE TITLE
Emgu.CV 4.4.0 with .Net Farmework 4.6.1

### ISSUE THREAD

**Issue by chrisn1983** [emgucv/emgucv#390]
Hi, recently tried switching over to Emgu 4.4.0 (from 4.3.0) for some opencv/neural net projects, and when I tried to compile an old program in .Net Framework 4.6.1 I was getting compile errors about not using .Net Core.  Is Emgu now incompatible with .Net Framework or is there some way to enable it?
Thanks (and sorry if this is a daft question, I usually write in C++ so a bit of a novice with .Net stuff)

**Comment by emgucv** [comment]
The Emgu.CV nuget package 4.4.0.4099 release is based on .NetStandard 2.0

If you look at the support matrix:
https://docs.microsoft.com/en-us/dotnet/standard/net-standard
.Net Standard 2.0 is compatible with .NetFramework 4.6.1 (with a foot note recommending to use .Net Framework 4.7.2 or higher).

The Motion Detection demo in the 4.4.0.4099 release is based on .NetFramework 4.6.1 and has referenced the Emgu.CV.Platform.NetStandard project, which targets .NetStandard 2.0. The project compiles and run without issues.

**Comment by chrisn1983** [comment]
Hi, I tried installing and compiling the Motion Detection demo you
suggested, and it compiles and runs with Framework 4.6.1, but when I use
the Nuget pacakage manager to get Emgu.CV.runtime.windows.cuda  4.4.0.4099
for my project, I get the following error.

[image: image.png]

Can you suggest any other properties or settings that I may have missed?

Thanks
Chris

**Comment by solarflarefx** [comment]
@chrisn1983 Were you able to find the root cause to your problem and fix the issue?

**Comment by chrisn1983** [comment]
> @chrisn1983 Were you able to find the root cause to your problem and fix the issue?

Hi @solarflarefx , sorry for the slow reply, took a while to find something that worked.  I eventually got it working by changing some NuGet options in Visual Studio (I'm using 2019) Tools > Options > NuGet Package Manager > General.  I cleared the cache ("Clear All NuGet Cache(s)" button) and changed the "Default package management format drop"  down to PackageReference - I've tried to attached the image below, not sure if it's uploaded correctly.

Hope it works for you too.

![NuGet settings](https://user-images.githubusercontent.com/61483875/112539421-d8f00c80-8e15-11eb-92ee-4092fabfb710.PNG)
